src/views/studio/CreateMemberView/CreateMemberView.tsx

```diff
@@ -133,10 +138,24 @@ const CreateMemberView = () => {
               required: true,
               pattern: MEMBERSHIP_NAME_PATTERN,
               patternMessage: 'may contain only lowercase letters, numbers and underscores',
+              validate: async (value) => {
+                // Wrapping it up with promise and resolving comparison inside debounce
+                // debounce() will not automatically do the return, which is needed for validation
+                return new Promise((resolve) => {
+                  debounce(async (handle) => {
+                    const {
+                      data: { membershipByUniqueInput },
+                    } = await refetchMember()
+                    resolve(membershipByUniqueInput?.handle !== handle)
```

Comment:
Was about to merge but just noticed this, I'm not sure if this check makes sense. We want to make sure that the membership with that handle doesn't exist right? So I'd expect
```suggestion
                    resolve(!membershipByUniqueInput)
```